src/pyscaffold/extensions/cirrus.py

```diff
@@ -39,7 +39,7 @@ def add_files(struct: Structure, opts: ScaffoldOpts) -> ActionParams:
     Returns:
         struct, opts: updated project representation and options
     """
-    files: Structure = {".cirrus.yml": (cirrus_descriptor, no_overwrite())}
+    files: Structure = {".cirrus.yaml": (cirrus_descriptor, no_overwrite())}
```

Comment:
Thanks for reviewing this PR so fast. I also think that `.readthedocs.yaml` and `docs/environment.yaml` should be added by default complementing the Sphinx setup that PyScaffold already provides. I created the issue #346.

I think the general consent is that every library asking for a yaml file should accept `.yml` and `.yaml` as file suffix. The [wikipedia article](https://en.wikipedia.org/wiki/YAML) even states  "The official recommended filename extension for YAML files has been .yaml since 2006". So I think it's meaningful to adapt this recommendation and Cirrus just doesn't seem to reflect that in their docs.

Reply:
Hi @FlorianWilhelm, thank you very much for the clarification. I can see where this is coming from more clearly now...

I understand that the official maintainers of YAML asked people to prefer `.yaml` over `.yml`, but I am afraid the community have not adopted that recommendation...

My feeling is that most tools using YAML configuration files officially document `.yml`. We can see that in Travis, AppVeyor, Cirrus CI. Even ReadTheDocs official documentation explicitly recommends `.yml` ([_"We recommend the filename .readthedocs.yml"_](https://docs.readthedocs.io/en/latest/config-file/v2.html#new-settings)), and Anaconda official guides/tutorial [uses a `.yml` in its examples](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file).

My concern is that newcomers might be a bit confused when they see the docs for all the tools using `.yml`.
I am not particularly attached to this, so I can go with either. But I can see the benefit on sticking with the officially documented file names.

If you agree, before merging the PR I can change the file names myself, so you don't have extra work :P.

Reply:
Hi @abravalheri, good points. I just thought it should be consistent no matter which option we go for. So it's `.yml` then. I'll change it and also open an issue in the dsproject-extension as there we have `environment.yaml` as far as I know.

Reply:
Okay, but `.pre-commit-config.yaml` should rather be `.yaml` as recommended from the docs. So then the rule of thumb is to use the suffix as recommended by the project even if it is inconsistent then :-/

Reply:
😞 that is unfortunate. But I think this is the least confusing thing to do...  At some point we might run into a tool that does not accept both extensions (I personally haven't tested Travis and GitlabCI with a `.yaml`, so I don't know if they would accept it).